lib/vscode/src/vs/server/node/server.ts

```diff
@@ -234,7 +234,7 @@ export class Vscode {
 		*/
 		const logService = new MultiplexLogService([
 			new ConsoleLogger(getLogLevel(environmentService)),
-			new SpdLogLogger(RemoteExtensionLogFileName, environmentService.logsPath, false, getLogLevel(environmentService))
+			new SpdLogLogger(RemoteExtensionLogFileName, path.join(environmentService.logsPath, "server.log"), false, getLogLevel(environmentService))
```

Comment:
```suggestion
			new SpdLogLogger(RemoteExtensionLogFileName, path.join(environmentService.logsPath, `${RemoteExtensionLogFileName}.log`), false, getLogLevel(environmentService))
```

Reply:
Nice suggestion! 🙌 Didn't realize that was supposed to be that!